src/servers/http_server.cc

```diff
@@ -1265,7 +1265,48 @@ HTTPAPIServer::HandleRepositoryControl(
         "'repository_name' specification is not supported");
   } else {
     if (action == "load") {
-      err = TRITONSERVER_ServerLoadModel(server_.get(), model_name.c_str());
+      struct evbuffer_iovec* v = nullptr;
+      int v_idx = 0;
+      int n = evbuffer_peek(req->buffer_in, -1, NULL, NULL, 0);
+      if (n > 0) {
+        v = static_cast<struct evbuffer_iovec*>(
+            alloca(sizeof(struct evbuffer_iovec) * n));
+        if (evbuffer_peek(req->buffer_in, -1, NULL, v, n) != n) {
+          HTTP_RESPOND_IF_ERR(
+              req, TRITONSERVER_ErrorNew(
+                       TRITONSERVER_ERROR_INTERNAL,
+                       "unexpected error getting load model request buffers"));
+        }
+      }
+      std::vector<const TRITONSERVER_Parameter*> params;
+      size_t buffer_len = evbuffer_get_length(req->buffer_in);
+      if (buffer_len > 0) {
+        triton::common::TritonJson::Value request;
+        HTTP_RESPOND_IF_ERR(
+            req, EVBufferToJson(&request, v, &v_idx, buffer_len, n));
+
+        // Parse request body for parameters
+        triton::common::TritonJson::Value param_json;
+        if (request.Find("parameters", &param_json)) {
+          triton::common::TritonJson::Value param;
+          if (param_json.Find("config", &param)) {
+            std::string config;
+            HTTP_RESPOND_IF_ERR(req, param.AsString(&config));
+            auto param = TRITONSERVER_ParameterNew(
+                "config", TRITONSERVER_PARAMETER_STRING, config.c_str());
+            if (param != nullptr) {
+              params.emplace_back(param);
+            } else {
+              HTTP_RESPOND_IF_ERR(
+                  req, TRITONSERVER_ErrorNew(
+                           TRITONSERVER_ERROR_INTERNAL,
+                           "unexpected error on creating Triton parameter"));
```

Comment:
Instead of "unexpected error on creating Triton parameter", maybe we can report a more verbose error?